common/general.d.ts

```diff
@@ -4,7 +4,7 @@ declare const enum SectionName {
     Leveling = "Leveling",
     Casting = "Casting",
 
-    Chapter3_Opening = "Chapter3_Opening",
+    Chapter3Opening = "Chapter3_Opening",
```

Comment:
Don't know about the naming for the enum, probably prefix all the enum entries by ChapterN (ie. add it for the Chapter1 ones too).